src/nodes/entity-config/editor/data/sensor.ts

```
@@ -146,7 +146,7 @@
     reactive_power: ['var'],
     signal_strength: ['dB', 'dBm'],
     sound_pressure: ['dB', 'dBA'],
-    speed: ['ft/s', 'in/d', 'in/h', 'km/h', 'kn', 'm/s', 'mm/d', 'mph'],
+    speed: ['ft/s', 'in/s', 'in/d', 'in/h', 'km/h', 'kn', 'm/s',' mm/s',' mm/h', 'mm/d', 'mph'],
```

ESLint

[error] Replace `'ft/s',·'in/s',·'in/d',·'in/h',·'km/h',·'kn',·'m/s','·mm/s','·mm/h',·'mm/d',·'mph'` with `⏎········'ft/s',⏎········'in/s',⏎········'in/d',⏎········'in/h',⏎········'km/h',⏎········'kn',⏎········'m/s',⏎········'·mm/s',⏎········'·mm/h',⏎········'mm/d',⏎········'mph',⏎····`

(prettier/prettier)